### PR TITLE
fix(ALP-217): Resolved odyssey owner icon issue

### PR DESCRIPTION
**Pull request by FIZLIN**
ALP-217

### CODE REVIEW

File: packages/app/src/scenes/widgets/Widgets.tsx [hunk 1]
```diff
@@ -58,7 +59,8 @@ const Widgets: FC<PropsInterface> = (props) => {
   useEffect(() => {
     onlineUsersStore.init(unityStore.worldId, sessionStore.userId);
     onlineUsersStore.fetchUser(unityStore.worldId);
-  }, [onlineUsersStore, sessionStore.userId, unityStore.worldId]);
+    odysseyBioStore.fetchUser(unityStore.worldId);
```
Comment (PAKovalev) [inline packages/app/src/scenes/widgets/Widgets.tsx]: Don't need to fetch before opening. Remove this line pls.

File: packages/app/src/scenes/widgets/Widgets.tsx [hunk 2]
```diff
@@ -165,13 +167,14 @@ const Widgets: FC<PropsInterface> = (props) => {
...
-                    icon={onlineUsersStore.nftUser?.avatarSrc ? undefined : 'people'}
+                    icon={nftUser?.avatarSrc ? undefined : 'people'}
                     size="medium"
                   >
-                    {onlineUsersStore.nftUser?.avatarSrc && (
+                    {nftUser?.avatarSrc && (
                       <Avatar
                         size="extra-small"
-                        avatarSrc={onlineUsersStore.nftUser?.avatarSrc}
+                        status={nftUser?.status}
```
Comment (PAKovalev) [inline packages/app/src/scenes/widgets/Widgets.tsx]: We don't have statuses for now at all.

File: packages/app/src/scenes/widgets/Widgets.tsx [hunk 3]
```diff
@@ -165,13 +167,14 @@ const Widgets: FC<PropsInterface> = (props) => {
...
+                        avatarSrc={nftUser?.avatarSrc}
```
Comment (PAKovalev) [inline packages/app/src/scenes/widgets/Widgets.tsx]: Make a view under UnityWorldStore with name `worldImageSrc`. See `isCurrentUserWorldAdmin` sample.

1. Nft has an image field.
2. In order to get an image url by hash use `getImageAbsoluteUrl` function.

Result should be `avatarSrc={unityWorldStore?.worldImageSrc}`

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: packages/app/src/scenes/widgets/Widgets.tsx [hunk 4]
```diff
@@ -90,13 +90,7 @@ const Widgets: FC<PropsInterface> = (props) => {
               title={t('titles.profile')}
               onClick={widgetsStore.profileStore.dialog.toggle}
             >
-              <Avatar
-                size="extra-small"
-                status={user?.status}
```
Comment (FIZLIN) [inline packages/app/src/scenes/widgets/Widgets.tsx:95]: Removed status from here as well. Is that okay, @PAKovalev?

**Review by PAKovalev — APPROVED** [review]

File: packages/app/src/scenes/widgets/Widgets.tsx (hunk 3 above)
Comment (FIZLIN) [reply packages/app/src/scenes/widgets/Widgets.tsx]: ✅

File: packages/app/src/scenes/widgets/Widgets.tsx (hunk 2 above)
Comment (FIZLIN) [reply packages/app/src/scenes/widgets/Widgets.tsx]: ✅

File: packages/app/src/scenes/widgets/Widgets.tsx (hunk 1 above)
Comment (FIZLIN) [reply packages/app/src/scenes/widgets/Widgets.tsx]: ✅

**Pull request merged**